ci/test/singularity

```diff
@@ -51,4 +51,8 @@ action "custom action" {
 EOF
 
 popper run
-popper run --quiet
\ No newline at end of file
+popper run --dry-run
```

Comment:
it would be best to implement checks that verify that providing these flags is having the desired effect. In the case of `--dry-run`, `--quiet` and `--debug`, we can leave them out. For `--reuse` and `--parallel`, however, it would be nice if we have tests. Since this might be significant work, we can create separate issues. Also, if we end up testing `--reuse` and `--parallel`, then they can be implemented in the corresponding test scripts (`test/parallel_stage_exec` and `test/reuse`).